ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentEItems/DocumentEItems.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Loader, Error, ResultsTable } from '@components';
+import { eitem as eItemApi } from '@api';
+import { BackOfficeRoutes } from '@routes/urls';
+import { SeeAllButton } from '@pages/backoffice/components/buttons';
+
+export default class DocumentEItems extends Component {
+  componentDidMount() {
+    this.props.fetchDocumentEItems(this.props.documentDetails.pid);
+  }
+
+  seeAllButton = () => {
+    const path = BackOfficeRoutes.eItemsListWithQuery(
+      eItemApi
+        .query()
+        .withDocPid(this.props.documentDetails.pid)
+        .qs()
+    );
+    return <SeeAllButton to={path} />;
+  };
+
+  viewDetails = ({ row }) => {
+    return (
+      <Link
+        to={BackOfficeRoutes.eitemDetailsFor(row.metadata.pid)}
+        data-test={row.metadata.pid}
+      >
+        {row.metadata.pid}
+      </Link>
+    );
+  };
+
+  filesFieldFormatter = ({ row }) => {
+    return row.metadata.files.length;
```

Comment:
What if it doesn't have any files?

Reply:
the object is always there, it will just show 0